spk/sickrage/src/installer.sh

```diff
@@ -26,8 +27,7 @@ syno_group_create ()
 {
     # Create syno group (Does nothing when syno group already exists)
     synogroup --add ${SYNO_GROUP} ${USER} > /dev/null
-    # Set description of the syno group
-    synogroup --descset ${SYNO_GROUP} "${SYNO_GROUP_DESC}"
+    # Set description of the syno group    synogroup --descset ${SYNO_GROUP} "${SYNO_GROUP_DESC}"
```

Comment:
seems like this happened when you fixed the last commit... Or is there any other reason why you disabled the synogroup description setting?